hadoop-hdds/docs/content/interface/CSI.zh.md

```diff
@@ -0,0 +1,78 @@
+---
+title: CSI 协议
+weight: 3
+summary: Ozone 支持 容器存储接口(CSI) 协议。你可以通过Ozone CSI 挂载 Ozone 桶的方式使用 Ozone。
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+容器存储接口 `Container Storage Interface` (CSI) 使存储供应商（SP）能够一次性开发一个插件，并让它跨多个容器编排工作。
+
+获取更多 CSI 的信息，可以参考[SCI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+
+Ozone CSI 是 CSI 的一种实现，它可以将 Ozone 用作容器的存储卷。 
+
+## 入门
+
+首先，我们需要一个带有 S3Gateway 的 Ozone 集群，并且它的 OM 和 S3Gateway 的端口都可以对 CSI pod 可见，
```

Comment:
In the original English doc, the `s3gateway` is used.

Reply:
Done. I should keep consistent with the English version, thanks, I've corrected it.